### ISSUE TITLE
Required gtk+ version 3.20, current version is 3.18

### ISSUE THREAD

**Issue by StephenDecatur** [lolstring/gnome-shell-extension-mycroft#15]
After I flipped the switched the switch on the GNOME Extensions page, Mycroft is disabled.  Error seems to be "Required gtk+ version 3.20, current version is 3.18".  Is there a stable and safe version that of gtk+ that I can update too, and how should I go about that? Or is there a way to lower the version of gtk+ that is required so that I can run Mycroft?

Running Mycroft on Ubuntu 16.04 GNOME Desktop

**Comment by DukhiAatma** [comment]
hi guys will this extension work on kubuntu ?

**Comment by GNOME-IS-LIFE** [comment]
nope, it will work on Gnome Shell. Gnome 3, while Kubuntu is Ubuntu with KDE Plasma. KDE Plasma already has an official Mycroft widget me thinks.

**Comment by GNOME-IS-LIFE** [comment]
StephenDecatur, I too, run Ubuntu 16.04 which comes with Gnome 3.18. There's nothing we can do but wait for 18.04 to come out with has 3.28 :smiley:
